packages/formstr-app/src/containers/CreateForm/components/FormSettings/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Divider, Switch, Typography } from "antd";
+import { DeleteOutlined } from "@ant-design/icons";
+import StyleWrapper from "./style";
+import useFormBuilderContext from "../../hooks/useFormBuilderContext";
+
+const { Text } = Typography;
+
+function FormSettings() {
+  const { formSettings, updateFormSetting } = useFormBuilderContext();
+
+  return (
+    <StyleWrapper>
+      <div className="form-setting">
+        <div className="property-setting">
+          <Text className="property-name">Title image</Text>
+          <Switch
```

Comment:
on toggled on, an input for image source should also be taken, which sets the banner image of the form, or do you think we should set this option somewhere on the image itself?  IMO an option in form settings  would be easier

Reply:
I think we should provide this option at both places. Over the image, we can provide an edit icon, so user can change the background from there